kie-wb-tests/kie-wb-tests-rest/src/main/java/org/kie/wb/test/rest/client/RestWorkbenchClient.java

```diff
@@ -356,19 +339,19 @@ public InstallProjectRequest installProject(String repositoryName, String projec
     }
```

Comment:
Method `public InstallProjectRequest installProject(String repositoryName, String projectName)` should be update. To use right path. `@Path("/organizationalunits/{organizationalUnitName}/projects/{projectName}/maven/install")`